common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt

```diff
@@ -72,4 +72,15 @@ interface INativeAuthCommandResult {
         open val correlationId: String = DiagnosticContext.INSTANCE.threadCorrelationId,
         open val errorCodes: List<Int>? = null
     )
+
+    data class InvalidUsername(
```

Comment:
Where is this class returned? I'm not seeing updates to the Controller or anywhere else.

Reply:
Updated